### ISSUE TITLE
Cannot find module ... /dist/ncc/sourcemap-register

### ISSUE THREAD

**Issue by ksmithut** [zeit/ncc#94]
I'm getting this error just trying to run a hello world example:

```
Error: Cannot find module '[redacted]/node_modules/@zeit/ncc/dist/dist/ncc/sourcemap-register'
```

I'm guessing it has something to do with the double `/dist/` in the path.

To run my quick example:

```sh
mkdir test
cd test
yarn add -D @zeit/ncc
echo "console.log('hello world')" > index.js
./node_modules/.bin/ncc run index.js
```

Versions:

```
> node --version
v10.14.1
> yarn --version
1.12.3
> cat package.json # this is of the package.json that get generated with the above commands
{
  "devDependencies": {
    "@zeit/ncc": "^0.2.0"
  }
}
```

**Comment by rauchg** [comment]
Apologize for this. Seems like the tests somehow didn't catch this.

**Comment by rauchg** [comment]
Fixed on `0.2.1` and `0.3.0`
